config/initializers/secure_headers.rb.rb

```diff
@@ -15,4 +15,10 @@
       (Rails.env.test? ? "'unsafe-inline'" : '')
     ]
   }
+
+  # So we can send JS errors to Sentry
+  # Strip off leading <pub_key>@ and trailing /<proj_num> so we have a clean
```

Comment:
Yes and it felt brittle - fell over if the http was missing - this seems more robust